### ISSUE TITLE
Setting the seed gives different results each time the code is run 

### ISSUE THREAD

**Issue by andrea-gatto** [Jack000/glid-3-xl#8]
When generating images setting the seed doesn't give reproducible results, the images (and clip scores) are different every time the code is run with same seed, any idea why?

**Comment by limiteinductive** [comment]
@andrea-gatto Yes, it is because some components of the diffusion process use non-deterministic algorithms like for example: https://github.com/pytorch/pytorch/issues/68961. There is no workaround for the moment.
